### PR TITLE
Update AUTHORS.txt and RELEASES.md

### PR DESCRIPTION
**Pull request by brson**
p=1 please

### CODE REVIEW

**Comment by brson** [comment]
[Randared](https://github.com/brson/rust/blob/relnotes/RELEASES.md)

File: AUTHORS.txt [hunk 1]
```diff
@@ -77,7 +85,7 @@ Ben Gamari <bgamari.foss@gmail.com>
 Ben Harris <mail@bharr.is>
 Ben Kelly <ben@wanderview.com>
 Ben Noordhuis <info@bnoordhuis.nl>
-Ben S <ogham@bsago.me>
+Ben S <ogham@users.noreply.github.com>
```
Comment (richo) [inline AUTHORS.txt]: Is this intentional?

Comment (brson) [inline AUTHORS.txt]: Yes. The bogus one is the most recent I have so I replaced the legit one.

File: RELEASES.md [hunk 2]
```diff
@@ -1,3 +1,102 @@
...
+    * [`Self` is a keyword][Self].
+
+* Libraries
+
+    * The `Show` and `String` formatting traits [have been
+      renamed][fmt] to `Debug` and `Display` to more clearly reflect
+      their related purposes. Automatically getting a string
+      conversion to use with `format!("{:?}", something_to_debug)` is
+      now written `#[derive(Debug)]`.
+    * Abstract [OS-specific string types][osstr], `std::ff::{OsString,
```
Comment (petrochenkov) [inline RELEASES.md]: typo: `ff` -> `ffi`

**Comment by jakub-** [comment]
I think we should also be updating `.mailmap` to keep it in sync.

**Pull request merged**